GVFS/GVFS/CommandLine/LogVerb.cs

```diff
@@ -109,11 +111,27 @@ private static string GetLogFilePatternForType(string logFileType)
             return "gvfs_" + logFileType + "_*.log";
         }
 
+        private static int MaxLogNameConsoleOutputWidth()

```

Comment:
I would consider naming this `GetMaxLogNameLength()`, since that's what we compute, even though we end up using to define a column width.